### ISSUE TITLE
Does not recognize !secret as valid

### ISSUE THREAD

**Issue by tacofart** [akfernun/yaml-sortkeys#1]
It does not seem to accept !secret as valid. 

For the following configuration.yaml:

```
ifttt:
  key: !secret ifttt_key
```

I get the error: 

```
unknown tag !<!secret> at line 3, column 29:
      latitude: !secret latitude
                                ^
```

I'm not sure if its an error specific to homeassistant but I dont get this validation error when using other yaml validation websites.

**Comment by akfernun** [comment]
Have you wrapped the value in quotes?

**Comment by tacofart** [comment]
No. That is not how !secret works for home assistant. Same error for !include

links:
https://home-assistant.io/topics/secrets/
https://home-assistant.io/topics/splitting_configuration/
